zerver/tests/test_home.py

```diff
@@ -221,11 +221,11 @@ def _sanity_check(self, result):
 
     def test_terms_of_service(self):
         # type: () -> None
-        email = 'hamlet@zulip.com'
+        user = self.example_user('hamlet')
+        email = user.email
         self.login(email)
 
         for user_tos_version in [None, '1.1', '2.0.3.4']:
-            user = get_user_profile_by_email(email)
             user.tos_version = user_tos_version
             user.save()
```

Comment:
I think this was doing things in a loop intentionally, though it happens to not really matter here since only one field gets mutated.  I don't see any problematic cases yet, but we should be sure to think about this sort of pattern while doing this migration...